thenewboston_node/accounts/views/node.py

```diff
@@ -35,3 +38,7 @@ def retrieve(self, request, pk: str = None):
 
         serializer = self.serializer_class(node)
         return Response(serializer.data)
+
+    def get_queryset(self):
+        blockchain = BlockchainBase.get_instance()
+        return SliceableCountableIterable(blockchain.yield_nodes(), count=blockchain.get_blockchain_states_count)
```

Comment:
copy paste issue: `get_nodes_count()`

Also, please, fix the unittest to avoid false-positives like this (the bug should have been caught by unittests)

Reply:
my bad, copy paste is the worst think, need to re-read 5 times when copy pasting :)